testCPUID-all/testCPUID.cpp

```diff
@@ -0,0 +1,433 @@
+
+#include <windows.h>
+#include <limits.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <intrin.h>
+#include <conio.h>
+#include <chrono>
+
+
+int g_proc_index = 0;
+
+const unsigned HS_CACHE_LINE_SIZE = 128;
```

Comment:
Do you mind writing few lines of `README.md` stating the purpose and usage?

Reply:
will do.